Newest Flake8 (3.8.3) is causing test failures E741

# Describe the bug

```
tests/test_user.py:620:13: E741 ambiguous variable name 'l'
tests/test_group.py:659:13: E741 ambiguous variable name 'l'
tests/test_course.py:1995:13: E741 ambiguous variable name 'l'
```

# To Reproduce

Upgrade to flake8 3.8.3
`pip install -I flake8`

Then run the tests
`flake8 canvasapi tests`

# Environment information

<!--(please complete the following information)-->

- 3.7.7 (`python --version`)
- develop (`pip show canvasapi`)

--- a/tests/test_course.py
+++ b/tests/test_course.py
@@ -1992,11 +1992,11 @@ class TestCourse(unittest.TestCase):
         self.assertIsInstance(licenses, PaginatedList)
         licenses = list(licenses)
 
-        for l in licenses:
-            self.assertIsInstance(l, License)
-            self.assertTrue(hasattr(l, "id"))
-            self.assertTrue(hasattr(l, "name"))
-            self.assertTrue(hasattr(l, "url"))
+        for lic in licenses:
+            self.assertIsInstance(lic, License)
+            self.assertTrue(hasattr(lic, "id"))
+            self.assertTrue(hasattr(lic, "name"))
+            self.assertTrue(hasattr(lic, "url"))
 
         self.assertEqual(2, len(licenses))
 

--- a/tests/test_group.py
+++ b/tests/test_group.py
@@ -656,11 +656,11 @@ class TestGroup(unittest.TestCase):
         self.assertIsInstance(licenses, PaginatedList)
         licenses = list(licenses)
 
-        for l in licenses:
-            self.assertIsInstance(l, License)
-            self.assertTrue(hasattr(l, "id"))
-            self.assertTrue(hasattr(l, "name"))
-            self.assertTrue(hasattr(l, "url"))
+        for lic in licenses:
+            self.assertIsInstance(lic, License)
+            self.assertTrue(hasattr(lic, "id"))
+            self.assertTrue(hasattr(lic, "name"))
+            self.assertTrue(hasattr(lic, "url"))
 
         self.assertEqual(2, len(licenses))
 

--- a/tests/test_user.py
+++ b/tests/test_user.py
@@ -617,11 +617,11 @@ class TestUser(unittest.TestCase):
         self.assertIsInstance(licenses, PaginatedList)
         licenses = list(licenses)
 
-        for l in licenses:
-            self.assertIsInstance(l, License)
-            self.assertTrue(hasattr(l, "id"))
-            self.assertTrue(hasattr(l, "name"))
-            self.assertTrue(hasattr(l, "url"))
+        for lic in licenses:
+            self.assertIsInstance(lic, License)
+            self.assertTrue(hasattr(lic, "id"))
+            self.assertTrue(hasattr(lic, "name"))
+            self.assertTrue(hasattr(lic, "url"))
 
         self.assertEqual(2, len(licenses))
 

